concurrent/src/test/java/io/airlift/concurrent/TestAsyncSemaphore.java

```diff
@@ -52,7 +52,7 @@ public void testInlineExecution()
 
         AtomicInteger count = new AtomicInteger();
 
-        List<ListenableFuture<?>> futures = new ArrayList<>();
+        List<ListenableFuture<Void>> futures = new ArrayList<>();
```

Comment:
Technically, none of these test changes are required, since this should be a backwards compatible change? (though we should update them)

Reply:
Yea, you don't need this, but it's what we would have done if we started with this pattern -- so might as well update it.